### PR TITLE
add description of loading a pretrained bundle

### PR DESCRIPTION
**Pull request by binliunls**
Signed-off-by: binliu <binliu@nvidia.com>

### Description
Add codes of how to load a pretrained bundlein this tutorial.

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

**Comment by binliunls** [comment]
Hi @Nic-Ma ,

I have added the description of how to load a pretrained bundle in this tutorial. Could you please help me to review it.

Thanks,
Bin

File: computer_assisted_intervention/video_seg.ipynb [hunk 1]
```diff
@@ -369,7 +369,14 @@
    "source": [
     "## Normal training loop\n",
     "\n",
-    "A normal training loop will be excuted. Meanwhile, the iou value of each epoch will be recorded as the accuracy."
+    "A normal training loop will be excuted. Meanwhile, the iou value of each epoch will be recorded as the accuracy. \n",
+    "There is a pretrained bundle for surgical tool segmentation task. This model can be loaded to the network by adding codes to the block below:\n",
```
Comment (Nic-Ma) [inline computer_assisted_intervention/video_seg.ipynb:373]: I would suggest to change it to:
"There is a pretrained bundle for surgical tool segmentation task in the MONAI model-zoo, the model can be automatically downloaded and loaded to the network by adding codes to the block below:"
What do you think?

Thanks.

File: computer_assisted_intervention/video_seg.ipynb [hunk 2]
```diff
@@ -369,7 +369,14 @@
...
+    "```\n",
+    "pretrained_model = monai.bundle.load(\n",
```
Comment (Nic-Ma) [inline computer_assisted_intervention/video_seg.ipynb:375]: Suggest var name "pretrained_model" to "pretrained_weights".

Thanks.

File: computer_assisted_intervention/video_seg.ipynb (hunk 1 above)
Comment (binliunls) [reply computer_assisted_intervention/video_seg.ipynb:373]: Hi @Nic-Ma ,
    I think it's ok and have updated the description.
Thanks,
Bin

File: computer_assisted_intervention/video_seg.ipynb (hunk 2 above)
Comment (binliunls) [reply computer_assisted_intervention/video_seg.ipynb:375]: Hi @Nic-Ma ,
    The var name has been updated.
Thanks,
Bin

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**